protocol/governance/cannonfile.ambassador-council.toml

```diff
@@ -0,0 +1,65 @@
+name = "synthetix-ambassador-council"
+description = "On-chain voting for synthetix Ambassador Council"
+version = "<%= package.version %>"
+
+[setting.salt]
```

Comment:
why aren't you `include`ing from the other `cannonfile.toml`? Doesn't that seem like it would result in signficantly less code duplication?